sdk/monitor/perf-tests/monitor-opentelemetry-exporter/README.md

```diff
@@ -0,0 +1,13 @@
+### Guide
+
+1. Build the monitor-opentelemetry-exporter perf tests package `rush build -t perf-monitor-opentelemetry-exporter`.
```

Comment:
Do you need to create a new project for this?, feel like this should be a subfolder in each project unless there is some shared stuff we want to have for all packages under monitor folder

Reply:
Followed the guidance here to conform to the same project directory structure as the other SDKs. https://github.com/Azure/azure-sdk-for-js/wiki/Writing-Performance-Tests#setting-up-the-project